src/main/scala/edu/ie3/simona/model/grid/GridModel.scala

```diff
@@ -545,6 +592,132 @@ case object GridModel {
     gridModel
   }
 
+  /** Build business models for transformer control groups
+    *
+    * @param config
+    *   List of configs for control groups
+    * @param measurementUnitInput
+    *   Set of [[MeasurementUnitInput]] s
+    * @return
+    *   A set of control group business models
+    */
+  private def buildTransformerControlGroups(
+      config: List[SimonaConfig.TransformerControlGroup],
+      measurementUnitInput: java.util.Set[MeasurementUnitInput],
+  ): Set[ControlGroupModel] = config.map {
+    case TransformerControlGroup(measurements, _, vMax, vMin) =>
```

Comment:
The control group is not connected to the configurated transformer yet. On the contrary, the configurated transformer is completely disregarded here. But I guess since nothing is happening yet here, this is not a big problem